### DISCUSSION TITLE
2023-07-31

### DISCUSSION THREAD

**Discussion by woo-jk** [woowacourse-teams/2023-haru-study#211] [daily_meeting]
#### 다같이
- 소켓, 폴링 양방향 통신 어떻게 할지
#### 백엔드
- 쿠키 관리 정책 재논의

### 언젠가 할 일
- 기획 변경된 이유 정리
- 질문 PlaceHolder로 유의미한 응답 예시를 제공하는 기능 (목표설정 가이드라인)
- 브랜치 전략 결정하기(dev -> main)
- A라는 스터디를 참여후 B스터디 참여한 뒤 다시 A스터디에 참여할 시 닉네임을 찾지 못한 이슈(로그인이 된다면 해결될 문제이긴 함)

**Comment by jaehee329** [comment]
### 체크인
7점 - 왜 월요일이죠?

### 체크아웃
5점 - 오늘 유의미하게 남은 결과가 없어서 기운이 떨어져요

**Comment by nlom0218** [comment]
### 체크인

9점 - 기분은 좋으나 시간이 부족하여 -1 😭😭😭

### 체크아웃

9점 - 여전히 시간이 부족하네요 흑흑 그래도 올만에 선릉갑니다 :)

**Comment by aak2075** [comment]
### 체크인

9점 - 어제 계곡가서 신나게 놀다왔지만 과음해서 조금 힘드네요

### 체크아웃

10점 - 팀원들과 많은 이야기를 나눠서 좋아요.

**Comment by MoonJeWoong** [comment]
### 체크인
9점 - 고민이 어느정도 해결되어서 기분은 좋은데 어제 늦게 잠들어서 -1점 깎았습니다!

### 체크아웃
9점 - 원래 생각했던 테이블 설계 변경 리팩토링은 잘 마무리해서 좋았는데 API 설계 변경에 대한 논의가 좀 남아서 9점입니당...

**Comment by woosung1223** [comment]
### 체크인
9점 - 별 생각이 없어서 9점입니다.

### 체크아웃
8점 - API 설계의 해답은 어디에 있을까..

**Comment by woo-jk** [comment]
### 체크인
8점 - 컨디션이 썩 좋지않다

### 체크아웃
8점 - 리팩토링을 예상보다 많이 하지 못했다

**Comment by yeopto** [comment]
### 체크인

10점 - 주말 푹 쉬었습니다

### 체크아웃

10점 - 스토리북 배포를 시원하게 끝내서 좋네요
